tests/cstor/gtest/test_zrepl_prot.cc

```diff
@@ -1828,6 +1828,55 @@ TEST(DiskReplaceTest, SpareReplacement) {
 	sleep(5);
 }
 
+static void verify_listsnap_details(std::string zvol_name) {
+	struct json_object *jobj = NULL, *jsnaplist_map = NULL;
+	struct json_object *jstrvolname, *jsnap = NULL;
+	std::string snapshot_output;
+	const char *volname, *snapname;
+	struct json_object_iterator it;
+	struct json_object_iterator itEnd;
+	std::string json, volsnap, snap;
+
+	json = execCmd("zfs", std::string("listsnap ") +
+	    zvol_name);
+
+	jobj = json_tokener_parse(json.c_str());
+	printf("%s\n", json.c_str());
+	ASSERT_NE((jobj == NULL), 1);
+
+	json_object_object_get_ex(jobj, "name", &jstrvolname);
+	volname = json_object_get_string(jstrvolname);
+	ASSERT_EQ(volname, zvol_name);
+
+	json_object_object_get_ex(jobj, "snaplist", &jsnaplist_map);
+
+	it = json_object_iter_begin(jsnaplist_map);
+	itEnd = json_object_iter_end(jsnaplist_map);
+
+	while (!json_object_iter_equal(&it, &itEnd)) {
+		snapname = json_object_iter_peek_name(&it);
+		snapshot_output = execCmd("zfs", std::string("list -t snapshot -Ho name " + zvol_name + std::string("@") + snapname));
+		printf("%s\n", snapshot_output.c_str());
```

Comment:
Let's avoid printing this value.

Reply:
its for debugging purpose in case of any issue.. any specific reason to avoid the print?

Reply:
If any error happened then it will be printed by the next line anyway.

Reply:
ok